lib/src/converter/protobuf_converter.dart

```diff
@@ -7,7 +7,7 @@ import 'package:flutter_reactive_ble/src/model/clear_gatt_cache_error.dart';
 import 'package:flutter_reactive_ble/src/model/connection_priority.dart';
 import 'package:flutter_reactive_ble/src/model/connection_state_update.dart';
 import 'package:flutter_reactive_ble/src/model/discovered_device.dart';
-import 'package:flutter_reactive_ble/src/model/discovered_services.dart';
+import 'package:flutter_reactive_ble/src/model/discovered_service.dart';
```

Comment:
just a small s maybe something to consider for us to revise to avoid confusion

Reply:
Likely we had some different classes (or different naming) there before we settled on the current implementation.